frontend2/src/content/bc22.ts

```diff
@@ -0,0 +1,68 @@
+import type {
+  CommonIssuesPageKey,
+  QuickStartedPageKey,
+  ResourcesPageKey,
+  TourneyPageKey,
+} from "./ContentStruct";
+
+export const BC22_TOURNAMENTS: Record<TourneyPageKey, string> = {
```

Comment:
The record type here is super cool and helps enforce that all fields are populated :star_struck: